cADFS.psm1

```diff
@@ -927,7 +927,7 @@ class cADFSNode {
         Write-Verbose -Message 'Starting retrieving ADFS Farm configuration.';
 
         try {
-            $AdfsProperties = Get-AdfsProperties -ErrorAction Stop;
+            Get-AdfsSyncProperties -ErrorAction Stop;
```

Comment:
Does this command exist on earlier versions (2012R2) of ADFS? I don't have a lab to spin one up and check how consistent it is between versions.